action.yml

```diff
@@ -94,9 +102,7 @@ runs:
       run: |
         source .tutor_venv/bin/activate
 
-        mkdir -p plugins
         cp $GITHUB_ACTION_PATH/patches.yml plugins/
-        tutor plugins enable patches
```

Comment:
Since we are not enabling the plugin in this step anymore, I think we should rename it
https://github.com/eduNEXT/integration-test-in-tutor/blob/f7766aaa8250737ddff7d3ed032a25c832cc917b/action.yml#L101

Reply:
Done: https://github.com/eduNEXT/integration-test-in-tutor/commit/e0e2168a410ceff814622955dc628d6f1be0cf4a